### PR TITLE
changes following voice-ai app setup

### PR DESCRIPTION
**Pull request by oliveranthony17**
Just some minor changes after working through the guide when setting up Voice AI - this app was a little different though as we tried to deploy with mrsk instead of Heroku.

Key changes:
- Added more explanation on Figaro gem
- I needed to add scss_lint gem - not sure if always required
- Note that additional rails command needed for simple_form
- Added some instruction for setting up Amazon S3

Still to do: the renuo-cli semaphore command creates files with the deprecated `--cache-folder` option. I'm happy to create a PR if agreed that changes needed.

### CODE REVIEW

File: ruby_on_rails/app_initialisation.md [hunk 1]
```diff
@@ -70,10 +70,12 @@ They are always idempotent (runnable multiple times).
...
 
 * Add `figaro` to Gemfile. Check the [gem homepage](https://github.com/laserlemon/figaro) to see how to install the gem
-(usually `bundle exec figaro install` is enough). Delete the newly created file `config/application.yml`.
+(usually `bundle exec figaro install` is enough). Delete the newly created file `config/application.yml`...
 * and create `config/application.example.yml` where you will specify the only environment variable you need for now:
   `SECRET_KEY_BASE`.
-* Add the following section to your `bin/setup` script so that the application.yml is created when the project is setup:
+* Going forward we will only push the `config/application.example.yml` file to the repository in order to protect our env variables.
+* Add application.yml to .gitignore
+* Add the following section to your `bin/setup` script so that the `application.yml` is created from the `application.example.yml` when the project is setup locally:
```
Comment (oliveranthony17) [inline ruby_on_rails/app_initialisation.md:78]: I felt this gem could do with a little more explanation for people who aren't familiar (me)

File: ruby_on_rails/app_initialisation.md [hunk 2]
```diff
@@ -82,8 +84,8 @@ They are always idempotent (runnable multiple times).
   end
   ```
 
-* add application.yml to .gitignore
```
Comment (oliveranthony17) [inline ruby_on_rails/app_initialisation.md:85]: moved this up

File: ruby_on_rails/configure_ci.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ and the main branches already pushed and ready to be tested.
 renuo configure-semaphore
 ```
 
-The command will copy the necessary templates to `.semaphore` folder.
+The command will copy the necessary templates to `.semaphore` folder using the renuo-cli. These files need to be maintained on the [renuo-cli repository](https://github.com/renuo/renuo-cli/tree/main).
```
Comment (oliveranthony17) [inline ruby_on_rails/configure_ci.md:20]: This actually needs amending now. In the semaphore.yml there are commands generated that use the `--cache-folder` option which is now deprecated

**Review by coorasse — APPROVED** [review]
Thanks!

**Review by ignaciosy — APPROVED** [review]
nice

**Review by rnestler — APPROVED** [review]
LGTM. I guess we can merge it?

**Pull request merged**